### PR TITLE
Hotfix/non functional changes

### PR DESCRIPTION
**Pull request by DeividasBakanas**

### CODE REVIEW

File: src/stores/map-store.ts [hunk 1]
```diff
@@ -219,18 +228,18 @@ export abstract class MapStore<TValue> extends ReduceStore<Items<TValue>> {
     /**
      * Check if given function is Promise.
      *
-     * @param calback {void | Promise<TResult>} function
+     * @param {any} [func] - function
      */
-    private isPromise<TResult>(func: any): func is Promise<TResult> {
-        return func != null && func.then != null;
+    private isPromise<TResult>(func?: any): func is Promise<TResult> {
```
Comment (GiedriusGrabauskas) [inline src/stores/map-store.ts]: It's required to give argument.

File: src/stores/map-store.ts [hunk 2]
```diff
@@ -250,17 +260,21 @@ export abstract class MapStore<TValue> extends ReduceStore<Items<TValue>> {
...
-     * @param key {string} Requested item key.
+     * @param {string[]} keys - Requested item key.
+     * @param {boolean} [noCache=false] - Update cached item from the server.
      */
-    public PrefetchAll(keys: Array<string>, noCache: boolean = false): void {
+    public PrefetchAll(keys: string[], noCache?: boolean): void
+    public PrefetchAll(keys: Immutable.List<string>, noCache?: boolean): void
+    public PrefetchAll(keys: Immutable.Set<string>, noCache?: boolean): void
+    public PrefetchAll(keys: Immutable.OrderedSet<string>, noCache?: boolean): void
+    public PrefetchAll(keys: any, noCache: boolean = false): void {
```
Comment (GiedriusGrabauskas) [inline src/stores/map-store.ts]: Missing semicolons in overload methods.

File: src/stores/map-store.ts [hunk 3]
```diff
@@ -306,19 +320,23 @@ export abstract class MapStore<TValue> extends ReduceStore<Items<TValue>> {
...
+     * @param {string[] | Immutable.List<string> } keys - Requested keys list in Array or Immutable List.
+     * @param {Immutable.Map<string, T>} [prev] - Previous data list merged with new data list.
+     * @param {boolean} [noCache=false] - Update cached items from the server.
+     * @return {Items<TValue>} Requested data list.
      */
+    public getAll(keys: string[], prev?: Items<TValue>, noCache?: boolean): Items<TValue>
+    public getAll(keys: Immutable.List<string>, prev?: Items<TValue>, noCache?: boolean): Items<TValue>
+    public getAll(keys: Immutable.Set<string>, prev?: Items<TValue>, noCache?: boolean): Items<TValue>
+    public getAll(keys: Immutable.OrderedSet<string>, prev?: Items<TValue>, noCache?: boolean): Items<TValue>
     public getAll(keys: any, prev?: Items<TValue>, noCache: boolean = false): Items<TValue> {
```
Comment (GiedriusGrabauskas) [inline src/stores/map-store.ts]: Missing semicolon in overloads

**Comment by DeividasBakanas** [comment]
Note: Check if all dist files were rebuilt before merge.

**Pull request merged**